src/Table/Table.less

```diff
@@ -0,0 +1,24 @@
+@import "../colors.less";
+@import "../fonts.less";
+
+.Table {
+  border: 1px solid @neutral_silver;
+  border-collapse: collapse;
+  color: @neutral_black;
+  font-family: "Proxima Nova";
+  font-size: 14px;
```

Comment:
hm, @CarlNelson -

0.75rem body looks too small:
![screen shot 2016-08-25 at 12 17 43 pm](https://cloud.githubusercontent.com/assets/16216084/17983173/20c08798-6ac0-11e6-8c4b-8c0dc0b15cdc.png)

and 1rem body looks a little too big:
![screen shot 2016-08-25 at 12 29 59 pm](https://cloud.githubusercontent.com/assets/16216084/17983184/2e8d96e0-6ac0-11e6-8d39-e2e995a705a2.png)

0.875rem (basically 14px) with 1rem header gets it looking sane, but 0.875 isn't currently available in design-system.
LMK if that's fine.